tests/test_actions_modernizer_django.py

```diff
@@ -0,0 +1,59 @@
+"""
+Tests for Github Actions Django Modernizer Script
+"""
+import os
+import shutil
+import uuid
+from unittest import TestCase
+
+from edx_repo_tools.codemods.django3 import GithubCIDjangoModernizer
+from edx_repo_tools.utils import YamlLoader
+
+
+class TestGithubActionsDjangoModernizer(TestCase):
+
+    def setUp(self):
+        self.test_file1 = self._setup_local_copy("sample_files/sample_ci_file.yml")
+        self.test_file2 = self._setup_local_copy("sample_files/sample_ci_file_2.yml")
+        self.test_file3 = self._setup_local_copy("sample_files/sample_ci_file_3.yml")
+
+    @staticmethod
+    def _setup_local_copy(file_name):
+        current_directory = os.path.dirname(__file__)
```

Comment:
we should instead use pytest's `tmpdir` to setup temporary copies of files as done here:
https://github.com/edx/repo-tools/blob/a9f47bdccb9a6731834954c2aeb195f3ccee9a87/tests/test_modernize_openedx_yaml.py#L9